docs/integrations/labext.md

```diff
@@ -0,0 +1,68 @@
+# Jupyter Lab Extension
+
+Now you can commit your Jupyter Notebook to [Jovian](https://jovian.ml?utm_source=docs) with just **One Click**.
+Make sure you’ve completed the [Installation](../user-guide/01-install.md) before reading further.
+
+## Using Jovian Jupyter Lab Extension
+
+<img src="https://i.imgur.com/uezsdYX.png" class="screenshot" alt="jupyter extension toolbar button">
+
+Once you have successfully installed jovian, a new button `Commit` will appear on the tool bar. When using `Commit` button for first time you'll be asked to provide an API key.
+
+<img src="https://i.imgur.com/jTvA0De.png" class="screenshot" alt="jovian api key prompt">
+
+You can get the API key at [Jovian](https://jovian.ml?utm_source=docs). Once you log in, just click on `API key` button, and the key will be copied to the clipboard.
+
+<img src="https://i.imgur.com/taLLUVd.png" class="screenshot" alt="jovian api key copy button">
+
+#### Valid API key
+
+If the key is valid you will be notified with the following alert.
+
+<img src="https://i.imgur.com/VYIvAk2.png" class="screenshot" alt="valid api key prompt">
+
+#### Error with API key
+
+If the entered API key is invalid you will get following error.
+
+<img src="https://i.imgur.com/Ot75GEP.png" class="screenshot" alt="api key error prompt">
+
+#### Successful Commit
+
+Once the API key has been validated, you can start committing to [Jovian](https://jovian.ml?utm_source=docs) by clicking `Commit` button. Once the Notebook has been committed successfully you will get the confirmation message with the link where the Jupyter Notebook has been uploaded to, you can click the link to your Notebook in Jovian.
+
+<img src="https://i.imgur.com/zxDHMA0.png" class="screenshot" alt="jovian commit success prompt">
+
+## Commit with more options
+
+This makes use of [jovian.commit's](../jvn/commit) parameters to enable the user to commit with preferences like private notebook, new notebook project, to add outputs and files .....
+
+**Step 1:** click the dropdown menu
+
+<img src="https://i.imgur.com/GUgZGcS.png" class="screenshot" alt="jovian dropdown menu">
+
+**Step 2:** choose `commit with options`
+
+<img src="https://i.imgur.com/NzRMRH8.png" class="screenshot" alt="jovian commit options jupyter extension">
+
+**Note:** By default the parameters are derived from [jovian.commit](https://jovian-py.readthedocs.io/en/latest/jvn/commit.html), changes to any parameter persists after commit.
+
+<img src="https://i.imgur.com/XdVkMPZ.png" class="screenshot" alt="jupyter extension commit options prompt">
+
+**Step 3:** Click on `Commit` to commit the notebook with following options.
+
+## Enable or Disable the extension
+
+By default, the Jovian Jupyter Lab Extension is enabled to the environment where jovian is installed.
+
+You can also disable the extension by running the following command.
+
+```
+$ jupyter labextension disable Jovian
```

Comment:
can you sync with mayank whether this works

Reply:
This command works to disable/enable the lab extension. I believe I just need to update the installation instruction.